### ISSUE TITLE
Callback on modal present

### ISSUE THREAD

**Issue by akshansh96** [gorhom/react-native-scroll-bottom-sheet#1078]
# Feature Request

It would be nice to have a callback once the modal is presented!

## Why it is needed

Modal takes some time to show up, I want a few actions to take place once that is done.

## Possible implementation

Referring to this: https://gorhom.github.io/react-native-bottom-sheet/modal/methods
`type present = () => void;` 
can change to 
`type present = () => ModalDidPresentCallback;`

### Code sample

WIP, I'll try to raise the PR by tomorrow..

**Comment by moseffect21** [comment]
+1
